CMakeLists.txt

```diff
@@ -256,8 +268,28 @@ install(
     DESTINATION "share/licenses/librdkafka"
 )
 
+INSTALL(FILES packaging/cmake/rdkafka.targets
+    DESTINATION "build/native"
```

Comment:
On Linux/Unix systems this installation destination (will be installed to `$PREFIX/build/native`) is not following the standard directory structure.
I'm not familiar with NuGet packages, but similar to Pkg-Config files or Cmake Modules, there should be a system-wide directory to install these files to.

Reply:
Nuget files are not deployment packages. They are used only at build time, so this should not be an issues.

Reply:
This has been solved. See comments in today's commits.